Configuration/DataProcessing/python/Impl/cosmicsEra_Run3_2021.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+"""
+_cosmicsEra_Run3_2021_
+
+Scenario supporting cosmic data taking
+
+"""
+
+import os
+import sys
+
+from Configuration.Eras.Era_Run3_cff import Run3
+from Configuration.DataProcessing.Impl.cosmics import cosmics
+
+class cosmicsEra_Run3_2021(cosmics):
```

Comment:
why 2021?
the scenarios in this directory should match exactly the era name at the end.